rackcity/views/asset_views.py

```diff
@@ -885,8 +951,7 @@ def asset_bulk_upload(request):
 
 
 @api_view(['POST'])
-@permission_required(PermissionPath.ASSET_WRITE.value, raise_exception=True)
-# TODO Check all assets for datacenter-level permissions
+@permission_classes([IsAuthenticated])
```

Comment:
Do you not check asset permission on bulk approve anymore?

Reply:
it's checked below. for assets that are added, and within the location validation method for assets that are modified

Reply:
oh sorry my b - it's not checked on asset / network connection bulk approve for the same reason we don't re-validate everything in approve modifications, any thing that's going to happen in approve should have been validated in bulk upload already